src/test/ui/consts/const-size_of-cycle.stderr

```diff
@@ -5,10 +5,6 @@ LL |     bytes: [u8; std::mem::size_of::<Foo>()]
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
 note: ...which requires const-evaluating `Foo::bytes::{{constant}}`...
-  --> $SRC_DIR/libcore/mem.rs:LL:COL
```

Comment:
this change seems wrong. Probably just local to your machine, should fail on travis

Reply:
I thought it was weird that `--bless` changed this file and as you expected it did fail on travis... any thoughts as to why it might have failed locally?

Reply:
I just remember seeing this in another PR.